### DISCUSSION TITLE
BW、BML快开票了，有没有能测试验证码的展子哇

### DISCUSSION THREAD

**Discussion by kevinlikun** [mikumifa/biliTickerBuy#165] [聊天专区]
我爬了一下getV2的allowPc字段
现在貌似没有限制手机下单的展子可以测试？

**Comment by kevinlikun** [comment]
发现BW和BML的allowPc字段是1，不知是不是因为没开票的原因（明日方舟和崩铁嘉年华现在看allowPc字段是0）

**Comment by mikumifa** [comment]
没有，要是有我就自己测试了

**Comment by mikumifa** [comment]
allowPc的值无所谓，抢票接口一样就可以了


验证码那个现在目前我没找到用来测试的，这就是我发公告的原因

今年五一我测试过是没问题的，但是不知道bw会不会突然做修改
